ActindoWorkFlow/Overview/02_WorkflowProcessElements.md

```diff
@@ -0,0 +1,194 @@
+
+![Input and output ports](/Assets/Screenshots/ActindoWorkFlow/Workflows/NewPlace.png "[Input and output ports]")
+
+All transitions contain a certain, predefined number of input and output ports, at least one of each. They are defined with a descriptive name, such as origin, data or destination.
+
+Input ports are positioned to the left of the transition and output ports are positioned to the right of the transition. Both are represented by a square.
+
+Both, input and output ports, can be of two types:
+
+- **Mandatory**  
+  Mandatory ports must be linked to a place, which in turn must be linked to a transition. Places linked to a mandatory port cannot be deleted or removed. When adding a new transition, all places linked to a mandatory input/output are already displayed.
+
+- **Optional**  
+ Optional ports are displayed with a ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button next to each of them. By clicking the ![Add](/Assets/Icons/Plus01.png "[Add]") (Add) button a new place is added, which can in turn be linked to a new transition. For detailed information about adding new actions, see [Create an action](/ActindoWorkFlow/Operation/01_ManageWorkflows.md#create-an-action).
+
+
+
+## Actions
+
+![Action](/Assets/Screenshots/ActindoWorkFlow/Workflows/AddAction.png "[Action]")
+
+Actions, or process actions, are the individual activities being executed in the workflow process and are represented by a rectangle.
+
+A process action is the actual execution of a transition  configured in a workflow. As such, a process action can be successful or faulty depending on its execution. If an error occurs, a process action can be retried if the underlying cause has been fixed, which results in a re-execution of the process action. Moreover, if multiple tokens move through the workflow, every token is processed in its own process action. As a result, a transition can be associated with multiple process actions, depending of the number of tokens running through it.
+
+
+
+## Tokens
+
+![Tokens](/Assets/Screenshots/ActindoWorkFlow/Workflows/Tokens.png "[Tokens]")
+
+
+A token is a container carrying data needed to execute an action, for example a number or a document.
+
+The *Processes* menu entry displays a diagram of any process taking place in real time and a list of all actions being executed in the selected process. For detailed information about tracking a workflow process, see [Track workflow process](/ActindoWorkFlow/Operation/02_TrackWorkflowProcess.md).
+
+A process is started with a single token at the start place, which will be input in an action to be executed. In turn, an action may output several tokens, which will become inputs for subsequent actions. It is therefore possible that a place contains several tokens, for example, several delivery notes issued for different parts of a same order. Each token will then be processed separately, that means, that the subsequent actions will be executed once for each token available.
```

Comment:
Hier nochmal deutlicher den Unterschied herausstellen:
1. mehrere Tokens können eine Action durchlaufen, z.B. mehrere delivery notes.
2. wenn ein Token eine Action durchläuft, können daraus mehrere, verschiedene Daten in unterschiedlichen output ports ausgegeben werden, sprich ein token rein, mehrere Tokens raus.